test/scale/flags.go

```diff
@@ -26,6 +26,7 @@ import (
 var (
 	skipClusterBootstrap = flag.Bool("skip-cluster-bootstrap", false, "flag to indicate that we can skip cluster bootstrap.")
 	scaleTestTimeout     = flag.Int("scale-test-timeout", 40, "flag to indicate the scale in and scale out test timeouts in minutes.")
+	testerReplica        = flag.Int("tester-replica", 2, "flag to indicate the number of replicas for the scale tests.")
```

Comment:
np: rename `testerReplica` to `replicas` or `replica-count`

Reply:
Sorry, I meant to say that we need to rename the command line flag name, so `tester-replica` to `replicas` or `replica-count` instead of `testerReplica `